fe/fe-core/src/main/java/org/apache/doris/common/util/LiteralUtils.java

```diff
@@ -32,7 +32,9 @@
 public class LiteralUtils {
 
     public static String getStringValue(FloatLiteral literal) {
-        if (literal.getType() == Type.TIME || literal.getType() == Type.TIMEV2) {
+        if (Double.isNaN(literal.getValue())) {
+            return "nan";
```

Comment:
we should not return nan. and this is not a right way to fix this problem. in sql, we could only return valid value of the type of this column or Null literal.

Reply:
if (Double.isNaN(literal.getValue())) {
            return Double.toString(literal.getValue());
        }
so changed?

Reply:
@morrySnow Do I change it as above, or do you change it?